src/Miunie.Discord/CommandModules/RemoteRepositoryModule.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading.Tasks;
+using Miunie.Core;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.CommandsNext;
+using DSharpPlus;
+using Miunie.Discord.Convertors;
+
+namespace Miunie.Discord.CommandModules
+{
+    public class RemoteRepositoryModule : BaseCommandModule
+    {
+        private readonly RemoteRepositoryService _remoteRepoService;
+        private readonly EntityConvertor _entityConvertor;
+
+        public RemoteRepositoryModule(RemoteRepositoryService remoteRepoService, EntityConvertor entityConvertor)
+        {
+            _remoteRepoService = remoteRepoService;
+            _entityConvertor = entityConvertor;
+        }
+
+        [Command("repo")]
+        [Description("Shows the official remote repository hosting the code of this bot")]
+        public async Task ShowRepository(CommandContext ctx)
+        {
+            var channel = _entityConvertor.ConvertChannel(ctx.Channel);
+            await _remoteRepoService.ShowRepository(channel);
+        }
+
+        [Command("setRepository")]
```

Comment:
This wasn't in the issue. The repository should come as a static resource from the LanguageProvider.